src/Symfony/Component/Scheduler/Trigger/CronExpressionTrigger.php

```diff
@@ -75,7 +75,7 @@ public static function fromSpec(string $expression = '* * * * *', string $contex
 
     public function getNextRunDate(\DateTimeImmutable $run): ?\DateTimeImmutable
     {
-        return \DateTimeImmutable::createFromMutable($this->expression->getNextRunDate($run));
+        return \DateTimeImmutable::createFromInterface($this->expression->getNextRunDate($run));
```

Comment:
This is a good change because it provides compatability if the dragonmantank/cron-expression package ever changes its return type. 

But why is a change to the Scheduler component included here?

Reply:
I'm moving this to a separate PR.